README.md

```diff
@@ -22,7 +22,8 @@
   <a href="./docs/README.MD">Documentation</a>
 </h3>
 
-<p align="center"><b>https://dogehouse.tv/</b></p>
+<p align="center">Current: <b>https://dogehouse.tv/</b></p>
+<p align="center">Beta: <b>https://next.dogehouse.tv/</b></p>
```

Comment:
I'd probably make them links rather than just urls.
Also, we can use this to keep the same style as the links above.

```suggestion
<p align="center">Current: <b>https://dogehouse.tv/</b></p>
<p align="center">Beta: <b>https://next.dogehouse.tv/</b></p>

<h4 align="center">
  <a href="https://dogehouse.tv">DogeHouse</a>
  <span> · </span>
  <a href="https://next.dogehouse.tv">Beta</a>
</h4>
```